package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
what is this file for?

Reply:
I have no idea. I assumed it was something poetry related, but now I'm realizing it was created in this PR. I definitely didn't make this on purpose lol I'll remove it.